index.html

```diff
@@ -138,6 +138,26 @@ <h2>CapturedMouseEvent interface</h2>
         more scrutiny in which information may safely be exposed. At present, only information
         already available to the capturing application is exposed - the mouse coordinates.
       </p>
+      <p>
+        If a <a data-cite="screen-capture/#dfn-capture-session">capture-session</a>
+        has an associated {{CaptureController}} whose
+        <a data-cite="screen-capture/#dfn-source">`[[Source]]`</a> is not
+        <a data-cite="mediacapture-streams/#track-ended">ended</a> then events with type
+        <dfn>capturedmousechange</dfn>, which do not bubble, which are not cancelable and which use
+        the {{CapturedMouseEvent}} interface, MUST be regularly dispatched on this
+        {{CaptureController}} to indicate any move of the captured cursor within the [=display
+        surface=] associated to that capture-session, or whether the captured cursor entered/exited
+        that surface.
```

Comment:
Could you please replace:
>```
> to indicate any move of the captured cursor within the [=display surface=]
> associated to that capture-session, or whether the captured cursor
> entered/exited that surface.
> ```
With:
>```
> to indicate the position of the captured cursor within the [=display surface=]
> associated with {{CaptureController/[[Source]]}}, or the cursor's departure from
> that surface.
> ```

Rationale:
1. If it's regularly dispatched and capped to N per second, we can't report every single coordinate update.
2. The cursor's reentry into the surface is already covered by reporting the coordinates within the surface.

Up for discussion:
* Whether "the surface associated with the capture-session" or "CaptureController.[[Source]]" is better for dynamic switching ([a](https://www.w3.org/TR/screen-capture/#dom-displaymediastreamoptions-surfaceswitching), [b](https://docs.google.com/presentation/d/1OLivIHYOLKjsFnINq2nibBk9RtwCQjkEyIazyiFzhh0/edit#slide=id.g22ae25380eb_1_0)).

Reply:
Thanks I'll do that. I was aware of 1 & 2 although I was thinking the note on 30times/seconds would disambiguate (1) ; while for (2) I was not sure whether "within" implied "stay inside" or if that includes entering.